ios/RnUnifiedPush.m

```diff
@@ -53,11 +59,10 @@ + (void)registerToUPS: (RCTResponseSenderBlock)callback {
   } success:^{
       NSLog(@"RN-IOS => UnifiedPush Server registration worked");
       NSLog(@"RN-IOS => Invoking callback");
-      callback(@[[NSNull null], @"Wow! Done!"]);
+      resolve(@[[NSNull null]]);
       NSLog(@"RN-IOS => Callback invoked");
   } failure:^(NSError * err) {
-    NSLog(@"RN-IOS => UnifiedPush Server registration Error: %@", err);
-    callback(@[err]);
+    reject( @"no_events", @"Error registering",err);
```

Comment:
For some reason the `err` object is not seen on the javascript side. The only message received is `Error registering`